modules/virt-importing-vm-prerequisites.adoc

```diff
@@ -50,6 +50,8 @@ Importing a VMware VM into {VirtProductName} has the following prerequisites:
 ====
 If you try to import a virtual machine with a disk that is larger than the available storage space, the operation cannot complete. You will not be able to import another virtual machine or to clean up the storage because there are insufficient resources to support object deletion. To resolve this situation, you must add more object storage devices to the storage backend.
 ====
+
+* Virtual disks must be connected to an IDE or SCSI controller. If a disk is connected to a SATA controller, you must change it to an IDE controller before migration. link:https://bugzilla.redhat.com/show_bug.cgi?id=1883588[(*BZ#1883588*)]
```

Comment:
Do we need the second sentence? The prereq seems to be fine as just '* Virtual disks must be connected to an IDE or SCSI controller.'
Also please remove the BZ link.

Reply:
The second sentence is the workaround. It does need to be there because the user needs to know that changing the controller from SATA to IDE will work for migration.

I can remove the BZ